api/src/wfl/module/aou.clj

```diff
@@ -134,7 +134,8 @@
       (make-labels per-sample-inputs other-labels))))
 
 (defn ^:private get-cromwell-environment! [{:keys [cromwell]}]
-  (let [envs (all/cromwell-environments #{:aou-dev :aou-prod} cromwell)]
+  (let [cromwell (all/de-slashify cromwell)
```

Comment:
do you need to generate a temporary here?

Reply:
The extra var here is only to make the error reporting accurate without inlining `(all/de-slashify cromwell)` twice. (for example, without it `https://unregistered-cromwell.broad.org/` throws `no unique environment matching Cromwell URL. {:cromwell https://unregistered-cromwell.broad.org/}`, the temporary makes it throw `no unique environment matching Cromwell URL. {:cromwell https://unregistered-cromwell.broad.org}` as the de-slashified url is indeed what WFL used for the filtering.)

I'm ok with either way, do you think moving `(all/de-slashify cromwell)` inline looks better? I can update it.

Reply:
The error reporting is something I hadn't thought about at all here. I don't really have any thoughts on this particular case partly because I think this class of error is something we'd want to solve at-scale when we start thinking about Terra integration and FISMA compliance. In other words, I think a lot of models for getting WFL FISMA-compliant are in conflict with the concept of an environments file, and I think solutions to that problem could be designed to eliminate this sorta class of errors and error messages.

Reply:
Ah yes errors! didn't see that - thanks.